mesosdriver/executor_driver_test.go

```diff
@@ -0,0 +1,54 @@
+package mesosdriver
+
+import (
+	"testing"
+
+	mesos "github.com/mesos/mesos-go/api/v1/lib"
+	"github.com/mesos/mesos-go/api/v1/lib/executor/config"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type MockDelegate struct{}
+
+func (m *MockDelegate) LaunchTask(taskInfo *mesos.TaskInfo) {}
+func (m *MockDelegate) KillTask(taskID *mesos.TaskID)       {}
+
```

Comment:
Without building real integration tests against a Mesos agent, this is as much as we can test. I may build those tests, but for now manual testing will stand in for that component.